rssab/library
# CSV Data Import Handler.

Writes feature to import CSV formatted catalog directly into the system. 

To test: 
- Ensure system is running in "dev" profile. 
- Add the --importcsvfile="/file/path/to/csv/file" argument to the system (either in IDE or as a command line argument while running). 
- Run the system and watch logs to verify file imported successfully.
